res/css/structures/_UserMenu.scss

```diff
@@ -208,17 +208,52 @@ limitations under the License.
 .mx_UserMenu_CustomStatusSection {
     margin: 0 12px 8px;
 
-    .mx_UserMenu_CustomStatusSection_input {
+    .mx_UserMenu_CustomStatusSection_field {
         position: relative;
         display: flex;
 
-        > input {
+        &.mx_UserMenu_CustomStatusSection_field_focused,
+        &.mx_UserMenu_CustomStatusSection_field_hasQuery {
+            .mx_UserMenu_CustomStatusSection_clear {
+                display: block;
+            }
+        }
+
+        > .mx_UserMenu_CustomStatusSection_input {
             border: 1px solid $accent;
             border-radius: 8px;
             width: 100%;
         }
+    
+        > .mx_UserMenu_CustomStatusSection_clear {
+            display: none;
+
+            position: absolute;
+            top: 50%;
+            right: 0;
+            transform: translateY(-50%);
+
+            width: 16px;
+            height: 16px;
+            margin-right: 8px;
+            background-color: $quinary-content;
+            border-radius: 50%;
+
+            &::before {
+                content: "";
+                position: absolute;
+                width: inherit;
+                height: inherit;
+                mask-image: url('$(res)/img/feather-customised/x.svg');
+                mask-position: center;
+                mask-size: 12px;
+                mask-repeat: no-repeat;
+                background-color: $secondary-content;
+            }
+        }
```

Comment:
These new styles are to support the "Clear" button which was in the DOM but an 0x0 invisible button.

These are copied from [`res/css/structures/_RoomSearch.scss#L40-L92`](https://github.com/matrix-org/matrix-react-sdk/blob/d00809b8b9023230c4c34b30f063288e1fe44608/res/css/structures/_RoomSearch.scss#L40-L92)